config/maps.txt

```diff
@@ -41,14 +41,11 @@ map cubestation
 	votable
 endmap
 
-map shitstation
```

Comment:
Just read this,
> (It cant be voted on, but can be rolled if Blob/Nukies/Another delta alert level threat, wins, in which case the whole server will cry and melt)

but it seems that this would mean runtime station and multiz_debug could both be rolled the same way which seems inaccurate and if there's some other way to prevent this we should be doing that rather than removing shit station.